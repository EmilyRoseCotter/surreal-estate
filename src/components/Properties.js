import React from "react";

function Properties() {
  return (
    <div>
      <h2>Properties</h2>
    </div>
  );
}

export default Properties;
