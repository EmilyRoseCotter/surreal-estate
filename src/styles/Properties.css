.properties-page {
  display: flex;
}

.alert .alert-error {
  margin: 5px;
}

.property-cards {
    display: flex;
    flex-wrap: wrap;
}

.column {
    display: inline-block;
    box-sizing: border-box;
    width: 33.33%;
    padding: 0.5rem;
}

@media only screen and (min-width: 1000px){
    .alert .alert-error {
    width: 20%;
  }
}

@media only screen and (max-width: 450px){
    .column {
    width: 100%;
  }
}

