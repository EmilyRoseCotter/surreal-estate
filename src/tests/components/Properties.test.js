import React from "react";
import { render } from "@testing-library/react";
import Properties from "../../components/Properties";

describe("View Properties page", () => {
  it("renders property cards", () => {
    const { asFragment } = render(<Properties />);

    expect(asFragment()).toMatchSnapshot();
  });
});
