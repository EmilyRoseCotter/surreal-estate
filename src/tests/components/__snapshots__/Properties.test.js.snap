// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View Properties page renders property cards 1`] = `
<DocumentFragment>
  <div
    class="properties-page"
  >
    <div
      class="alert"
    />
    <div>
      <div
        class="property-cards"
      />
    </div>
  </div>
</DocumentFragment>
`;
